### PR TITLE
Adds missing shows and some german synonyms

### PR DESCRIPTION
**Pull request by InteractiveNinja**
Added shows:
* My Next Life as a Villainess [S1](https://anilist.co/anime/104647/Otome-Game-no-Hametsu-Flag-shika-Nai-Akuyaku-Reijou-ni-Tensei-shiteshimatta/) [S2](https://anilist.co/anime/120209/Otome-Game-no-Hametsu-Flag-shika-Nai-Akuyaku-Reijou-ni-Tensei-shiteshimatta-X/)
* Anti-Magic Academy: The 35th Test Platoon [S1](https://anilist.co/anime/20704/Taimadou-Gakuen-35-Shiken-Shoutai/)
* Arifureta: From Commonplace to World's Strongest [S1](https://anilist.co/anime/100668/Arifureta-Shokugyou-de-Sekai-Saikyou/) [S2](https://anilist.co/anime/112323/Arifureta-Shokugyou-de-Sekai-Saikyou-2nd-season/)
* Summer Time Rendering [S1](https://anilist.co/anime/129201/Summer-Time-Render/)
* Suppose a Kid from the Last Dungeon Boonies moved to a starter town? [S1](https://anilist.co/anime/112649/Tatoeba-Last-Dungeon-Mae-no-Mura-no-Shounen-ga-Joban-no-Machi-de-Kurasu-Youna-Monogatari/)
* GRANBLUE FANTASY The Animation [S1](https://anilist.co/anime/21361/GRANBLUE-FANTASY-The-Animation/) [S2](https://anilist.co/anime/107346/GRANBLUE-FANTASY-The-Animation-Season-2/)
* The Strongest Sage with the Weakest Crest [S1](https://anilist.co/anime/129191/Shikkakumon-no-Saikyou-Kenja/)

Added Synoyms per Show:
**Is It Wrong to Try to Pick Up Girls in a Dungeon?**
* Danmachi: Is It Wrong to Try to Pick Up Girls in a Dungeon?
* Danmachi

**The Slime Diaries: That Time I Got Reincarnated as a Slime**
* Die Schleim-Tagebücher (German Title)

**That Time I Got Reincarnated as a Slime**
* Meine Wiedergeburt als Schleim in einer anderen Welt (German Title)

**Chaika the Coffin Princess**
* Chaika - Die Sargprinzessin (German Title)

### CODE REVIEW

**Comment by InteractiveNinja** [comment]
> @reconman just saw the commit from my last PR [af9a242](https://github.com/RickDB/PlexAniSync-Custom-Mappings/commit/af9a242997a3bc75ce3a7ef81c1ea3866d6d99f4) Is there an order that should be followed?

nvm just saw its alphabetically..

File: series-tvdb.en.yaml [hunk 1]
```diff
@@ -51,6 +51,16 @@ entries:
         anilist-id: 20853
         start: 13
 
+  - title: "Arifureta Shokugyou de Sekai Saikyou"
```
Comment (reconman) [inline series-tvdb.en.yaml:54]: English titles as main, Japanese as synonym

File: series-tvdb.en.yaml [hunk 2]
```diff
@@ -1080,6 +1103,19 @@ entries:
       - season: 1
         anilist-id: 21087
 
+  - title: "Otome Game no Hametsu Flag shika Nai Akuyaku Reijou ni Tensei shiteshimatta…"
```
Comment (reconman) [inline series-tvdb.en.yaml:1106]: Same

File: series-tvdb.en.yaml [hunk 3]
```diff
@@ -1146,6 +1182,13 @@ entries:
       - season: 1
         anilist-id: 127911
 
+  - title: "Shikkakumon no Saikyou Kenja"
```
Comment (reconman) [inline series-tvdb.en.yaml:1185]: Same

File: series-tvdb.en.yaml [hunk 4]
```diff
@@ -1146,6 +1182,13 @@ entries:
...
+    synonyms:
+      - "The Strongest Sage with the Weakest Crest"
+      - "Der stärkste Held mit dem Mal der Schwäche"
+    seasons:
+      - season: 1
+        anilist-id: 129191
```
Comment (reconman) [inline series-tvdb.en.yaml:1191]: Missing empty line

File: series-tvdb.en.yaml [hunk 5]
```diff
@@ -1242,9 +1293,26 @@ entries:
       - season: 5
         anilist-id: 21673
 
+  - title: "Taimadou Gakuen 35 Shiken Shoutai"
```
Comment (reconman) [inline series-tvdb.en.yaml:1296]: Same

File: series-tvdb.en.yaml [hunk 6]
```diff
@@ -1242,9 +1293,26 @@ entries:
...
+    synonyms:
+      - "Anti-Magic Academy: The 35th Test Platoon"
+      - "Anti Magic Academy: Test-Trupp 35"
+    seasons:
+      - season: 1
+        anilist-id: 20704
+
+  - title: "Tatoeba Last Dungeon Mae no Mura no Shounen ga Joban no Machi de Kurasu Youna Monogatari"
```
Comment (reconman) [inline series-tvdb.en.yaml:1304]: Same

**Review by reconman — CHANGES_REQUESTED** [review]

**Comment by reconman** [comment]
Closed via 77803b332b2d64829a33446ad6e81f7101846fc6

**Pull request closed without merging**